service/src/main/java/com/ge/predix/acs/audit/PredixEventProcessor.java

```diff
@@ -1,34 +1,63 @@
 package com.ge.predix.acs.audit;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.Set;
+
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
 
 import com.ge.predix.audit.AuditEvent;
 import com.ge.predix.audit.AuditEventProcessor;
 import com.ge.predix.audit.sdk.AuditClient;
-import com.ge.predix.audit.sdk.exception.AuditException;
 import com.ge.predix.audit.sdk.message.AuditEventV2;
 
+@Component
+@Profile("predixAudit")
 public class PredixEventProcessor implements AuditEventProcessor {
 
     private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PredixEventProcessor.class);
 
-    @Autowired
-    private AuditClient auditClient;
+    // Remove this and use configuration eventually
+    private static final Set<String> EXCLUSIONS = Collections.singleton("/connector");
 
     @Autowired
     private PredixEventMapper eventMapper;
 
+    @Autowired
+    private AuditClient auditClient;
+
     @Override
     public boolean process(final AuditEvent auditEvent) {
-        AuditEventV2 predixEvent = eventMapper.map(auditEvent);
+        if (null == this.auditClient || isExclusion(auditEvent)) {
```

Comment:
The `null == this.auditClient` condition is redundant since it will always be false (since the `predixAudit` Spring profile always wires in the `auditClient` bean).